tests/Tests.AzureAppConfiguration/RefreshTests.cs

```diff
@@ -1132,6 +1127,42 @@ public void RefreshTests_UpdateCacheExpirationTimeForFailedRefreshOperations()
             Assert.NotEqual("newValue", config["TestKey1"]);
         }
 
+#if NET7_0
+        [Fact]
+        public void RefreshTests_ChainedConfigurationProviderUsedAsRootForRefresherProvider()
+        {
+            var chainedConfigurationProvider = new ChainedConfigurationSource
+            {
+                Configuration = new ConfigurationBuilder()
+                .Add(new ChainedConfigurationSource
+                {
+                    Configuration = new ConfigurationBuilder()
+                        .AddAzureAppConfiguration(optionsInitializer, optional: true)
+                        .Build()
+                })
+                .Build()
+            }
+            .Build(new ConfigurationBuilder()) as ChainedConfigurationProvider;
+
+            PropertyInfo propertyInfo = typeof(ChainedConfigurationProvider).GetProperty("Configuration", BindingFlags.Public | BindingFlags.Instance);
```

Comment:
I dont think we need reflection in unit tests. We can test the following:
- Create IConfiguration that loads data from AppConfig.
- Create another IConfiguration that loads data using the previous config. 
- Now create  AzureAppConfigurationRefresherProvider with the second config. This refresher provider should have a single refresher.